_posts/2016-04-27-discounted-goruco-tickets-for-underrepresented-people.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+excerpt_separator: <!--more-->
+---
+
+<p>
+  This year we are offering discounted tickets for those who self-identify
```

Comment:
"self-identify" -> "identify"

As I was reading it, I think "self-identify" is redundant.